tasks/95_pairwise/pairwise.js

```diff
@@ -0,0 +1,12 @@
+export function pairwise(arr, num) {
+  return arr.reduce((sum, item, index) => {
+    for (let i = index + 1; i < arr.length; i++) {
+      if (item + arr[i] === num) {
+        arr[index] = '';
```

Comment:
what is a reason to reset current value? you will not return to this value anyway